cs/unittest/RunTests.tt

```diff
@@ -22,7 +22,7 @@ var skipList = new[] { 13, 32, 39, 40, 41, 59, 60, 61, 66, 68, 90,
 	14, 16, 17, 31, 33, 34,53, 101, 102, 103, 105, 106, 111, 112, // float delta
 	71, // --examples to test parser
 	143, 144, 146, 158, 189, // native json parsing 
-	149, 152, 156, // bash script
+	149, 152, 156, 191, 192, // bash script
 	188 // possibly float delta
```

Comment:
Given that --onethread doesn't work in the windows tests, we should just add an exception for 193 as well.

Reply:
I would put it under a different category, though: For example, "executable-only feature used".